packages/hydrogen-edge-adapter/README.md

```diff
@@ -0,0 +1,57 @@
+## Hydrogen Remix Adapter for Netlify
+
+The Hydrogen Remix Adapter for Netlify allows you to deploy your [Shopify Hydrogen](https://hydrogen.shopify.dev/) (via
+[Remix](https://remix.run)) app to [Netlify Edge Functions](https://docs.netlify.com/edge-functions/overview/).
+
+### Usage
```

Comment:
```suggestion
# Hydrogen Remix Adapter for Netlify

The Hydrogen Remix Adapter for Netlify allows you to deploy your [Shopify Hydrogen](https://hydrogen.shopify.dev/) (via
[Remix](https://remix.run)) app to [Netlify Edge Functions](https://docs.netlify.com/edge-functions/overview/).

## Usage
```

I typically follow the heading structure starting with h1 as those READMEs will also be displayed on npm or other places and then it's correct everywhere